include/tvm/relay/transform.h

```diff
@@ -83,18 +117,81 @@ class PassContextNode : public RelayNode {
    */
   ErrorReporter err_reporter;
 
+  /*! \brief The default optimization level. */
+  int opt_level{2};
+
+  /*! \brief CPU is the default fallback device for heterogeneous execution. */
+  int fallback_device{static_cast<int>(kDLCPU)};
+
+  /*! \brief The list of required passes. */
+  tvm::Array<tvm::Expr> required_pass;
+  /*! \brief The list of disabled passes. */
+  tvm::Array<tvm::Expr> disabled_pass;
+
+  /*! 
+   * \brief A helper struct to get the optimization pass name to opt level
+   * mapping.
+   */
+  OptPassLevel OPT_PASS_LEVEL;
```

Comment:
remove this from the context as it is global state and is only needed in Sequential